ui/app/src/hooks/useCore.ts

```diff
@@ -17,6 +17,13 @@ const store = createStore();
 const actions = createActions({ store, api });
 const poolFinder = createPoolFinder(store);
 
+// expose store on the window so it is easy to inspect
+Object.defineProperty(window, "store", {
```

Comment:
Should we remove this part when done testing?

Reply:
I think it is useful to be able to inspect this generally and will help with isolating some of these UI bugs. There should be no sensitive information here and you cannot write to the store.

Reply:
I agree. Let's keep for now

Reply:
By the way I tried exposing it for Vue devtools on the App component but that chugged big time and this seemed to be the better way to go.